docs/docs-content/release-notes.md

```diff
@@ -11,155 +11,120 @@ sidebar_custom_props:
+- Automatic SSL certificate renewal is now supported for clusters deployed through Palette. In the past, this was a manual action that had to be performed by the user, which also caused node repaves. Palette will automatically renew the certificate 30 days before the expiration date without triggering a node repave. This feature is available in all supported infrastructure providers except for Edge and Cox Edge. For more information, refer to the [Certificate Management](./clusters/cluster-management/certificate-management.md) resource.
 
 
-- Self-hosted Palette installations, and Private Cloud Gateway (PCG), now support the ability to control the node affinity for the internal Palette components for VMware vSphere target clusters. You can configure the Palette Enterprise, or PCG cluster to deploy all internal component pods on control plane nodes. This is an option you enable during the install. 
+- Enhanced cluster repave control and mitigation. In the Palette 4.1 release, repave notification warnings become available through the User Console (UI). In this release, cluster administrators, project administrators, and tenant administrators must acknowledge the repave notification and decide whether to proceed with the action. This feature helps prevent accidental repaves and provides a way to mitigate repaves by allowing administrators to cancel the repave action.
 
 
-- Palette, in a multi-SaaS environment, has been updated with a new feature that allows you and your team to receive platform notifications. This feature will keep you informed of any upcoming changes, news, or releases related to Palette. Additionally, you can now access product information and more through the new widget located in the bottom right corner of the Palette user interface. 
+- Palette CLI now supports integration with [Validator](https://github.com/spectrocloud-labs/validator), an open-source framework that you can use to validate your environment. Validator performs Day 0-2 validation and configuration drift detection in a composable manner across various systems. Use the `palette validator` command to verify your environment before installing a self-hosted instance of Palette or VerteX. You can also use Validator to verify the environment requirements for deploying a cluster. For more information, refer to the [Validator](./palette-cli/commands/validator.md) CLI reference.
 
 
-- The enhanced pack experience introduced in Palette 4.0.0 for add-on packs is now available to all packs.
+- Support for passkeys is now available for the self-hosted Palette admin user. When accessing the system console, you can now use passkeys to authenticate to the admin user account. For more information, refer to the [System Console Credentials](./enterprise-version/system-management/account-management/credentials.md) resource.
```

Comment:
@karl-cardenas-coding This is in the VerteX section, which may be a bit confusing. Although it was driven by federal requirements, it's applicable to Palette and VerteX AFAIK.

Reply:
@cloudmaniac I asked in an Engineering weekly two weeks ago, and I was told it was for both Palette and VerteX.

Reply:
@cloudmaniac that link is for the Palette section. VerteX has its own section with this release notes that contains a link to its own section.